Plugins~/Src/MeshSyncClientBlender/python/3.1.0/unity_mesh_sync.py

```diff
@@ -128,6 +128,12 @@ def __del__(self):
     def execute(self, context):
         return self.invoke(context, None)
 
+    # When a file is loaded, the modal registration is reset:
+    @persistent
+    def load_handler(dummy):
+        MESHSYNC_OT_AutoSync._registered = False
+        bpy.app.handlers.load_post.remove(MESHSYNC_OT_AutoSync.load_handler)
```

Comment:
Is this related to this PR or an independent bug fix?

Reply:
Unrelated to this feature, I've created a separate PR for this here: https://github.com/Unity-Technologies/MeshSyncDCCPlugins/pull/209